### ISSUE TITLE
graalvm-ce-java11_amd64_21.0.0.2-0.deb does not install

### ISSUE THREAD

**Issue by egonw** [dongjinleekr/graalvm-ce-deb#5]
I get this error:

```shell
dpkg-deb: error: archive 'graalvm-ce-java11_amd64_21.0.0.2-0.deb' uses unknown compression for member 'control.tar.zst', giving up
```

**Comment by dongjinleekr** [comment]
Which dpkg version are you using? graalvm-ce-deb is [compressed with zstd](https://lists.ubuntu.com/archives/ubuntu-devel/2018-March/040211.html).

**Comment by egonw** [comment]
https://packages.debian.org/buster/dpkg (1.19.7)

**Comment by dongjinleekr** [comment]
Wait... then, you are using Debian, not Ubuntu. right?

- [dpkg (debian/buster)](https://packages.debian.org/buster/dpkg)
- [dpkg (ubuntu/bionic)](https://packages.ubuntu.com/bionic/dpkg)

Reviewing dependencies, it seems like zstd compression is not supported in upstream dpkg yet. 😓

**Comment by dongjinleekr** [comment]
Okay, for Debian users, it would be better to revert the compression codec into gzip.

**Comment by dongjinleekr** [comment]
Resolved by 2fbee10, [release 21.0.0.2-1](https://github.com/dongjinleekr/graalvm-ce-deb/releases/tag/21.0.0.2-1).
